### PR TITLE
Quality of life changes to runsim

### PR DESCRIPTION
**Pull request by mircea-c**
- [x] SQS check to coordinate with other sim instances
- [x] Refactor log push function

### CODE REVIEW

File: cmd/runsim/main.go [hunk 1]
```diff
@@ -97,6 +90,13 @@ Run simulations in parallel`, filepath.Base(os.Args[0]))
 func main() {
 	var err error
 
+	runsimLogfile, err := os.OpenFile("sim_log_file", os.O_RDWR|os.O_APPEND|os.O_CREATE, 0666)
+	if err != nil {
```
Comment (alessio) [inline cmd/runsim/main.go:94]: we should just fail here:

```golang
log.Fatalf("couldn't open sim_log_file: %v", err)
```

File: cmd/runsim/main.go [hunk 2]
```diff
@@ -97,6 +90,13 @@ Run simulations in parallel`, filepath.Base(os.Args[0]))
...
+		log.Printf("ERROR: opening log file: %v", err.Error())
```
Comment (alessio) [inline cmd/runsim/main.go]:
```suggestion
		log.Fatalf("couldn't open sim_log_file: %v", err)
```

File: cmd/runsim/main.go [hunk 3]
```diff
@@ -97,6 +90,13 @@ Run simulations in parallel`, filepath.Base(os.Args[0]))
...
+	} else {
```
Comment (alessio) [inline cmd/runsim/main.go]: If above suggestion is accepted, this should go

File: cmd/runsim/notification.go [hunk 4]
```diff
@@ -2,82 +2,101 @@ package main
 
 import (
 	"fmt"
+	"github.com/pkg/errors"
```
Comment (alessio) [inline cmd/runsim/notification.go]: this makes sense if you intend to use the `Wrap()` function.
Alternatively I'd suggest to depend on the standard `errors` package.

Comment (mircea-c) [reply cmd/runsim/notification.go]: oops, that was just Goland auto importing stuff. Fixed.

**Review by alessio — APPROVED** [review]

**Pull request merged**